src/passkey_child/passkey_child.h

```diff
@@ -339,13 +342,15 @@ select_authenticator(struct passkey_data *data, fido_dev_t **_dev,
 /**
  * @brief Set client data hash in the assert
  *
+ * @param[in] data passkey data
  * @param[out] assert Assert
```

Comment:
In both cases it's an input parameter

Reply:
You pass in existing assert and then you write to it with `fido_assert_set_clientdata_hash`. The doc string is certainly not clear since it suggest that _assert is only out parameter.